### DISCUSSION TITLE
Data Export of Users

### DISCUSSION THREAD

**Discussion by benjibee** [stuttter/wp-user-groups#33] [General]
Not sure if this is indeed an issue, or just a feature request, or neither, but we have the following issue using this plugin:

So now we've got hundreds of users, and we'd like to be able to export them to a CSV or Excel file. We're using Admin Columns Pro but it looks like it can't see the user groups created with this plugin, so we tried WP All Export and that too doesn't seem to see the users groups. Any idea how to export the user data with groups?

Thanks!

**Comment by JJJ** [comment]
Under the hood, Groups are Taxonomies, and the object IDs are user IDs.
